### PR TITLE
Add missing protobuf definitions

### PR DESCRIPTION
**Pull request by penso**
## What is the purpose of the change

> Add a description of the overall background and high level changes that this PR introduces

This is adding missing protobuf definitions preventing [Constellations](https://www.constellations.zone) from indexing, which would prevent anyone else as well.

## Testing and Verifying

This change is a trivial rework / code cleanup without any test coverage.

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes?
  - [X] Changelog entry added to `Unreleased` section of `CHANGELOG.md`?

Where is the change documented?
  - [ ] Specification (`x/{module}/README.md`)
  - [ ] Osmosis documentation site
  - [ ] Code comments?
  - [X] N/A

### CODE REVIEW

**Review by mattverse — COMMENTED** [review]
Is the issue that / and are these proto additions the ones that we have had in the past blocks but have been deleted thus cannot be queried / indexed?

**Comment by penso** [comment]
> Is the issue that / and are these proto additions the ones that we have had in the past blocks but have been deleted thus cannot be queried / indexed?

Yes, those proto messages are on chain, but no proto definitions allowing to decode them. Adding those definitions from this PR I was able to index old blocks, outside a few which I think never got protobuf files committed anywhere.

File: CHANGELOG.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Bug Fixes
+
+* Add missing protobuf definitions to be able to decode old block messages
```
Comment (czarcas7ic) [inline CHANGELOG.md:47]: Please add PR number to the changelog

**Comment by czarcas7ic** [comment]
@penso thanks for this! A couples points / questions

1. I don't understand why the bridge folder needs to come back. There were zero messages on chain that ever used any of these proto definitions, so it should not be required for any indexing.
2. For the others, can we add some kind of copy pasta comment above each of these, saying XXX method is no longer used, but this is kept here for the purposes of indexing?

**Comment by mattverse** [comment]
Also, @penso Did you ever get to try querying old state with these added? Did it work? 👀

**Comment by PaddyMc** [comment]
friendly ping here @penso, see: https://github.com/osmosis-labs/osmosis/pull/8600#issuecomment-2274913621

**Comment by penso** [comment]
Ah yeah a few things to fix, will do them today.

**Comment by czarcas7ic** [comment]
@penso thanks for addressing these!

@mattverse if you have time to give this a once over, if it looks good to you I think we are good to merge.

**Review by mattverse — APPROVED** [review]
This LGTM

**Comment by gilxgil** [comment]
Thank you @faboweb we've been waiting for this patch for so long! https://github.com/osmosis-labs/osmosis/issues/7675

**Comment by gilxgil** [comment]
Thank you @penso we've been waiting for this patch for so long! https://github.com/osmosis-labs/osmosis/issues/7675

**Pull request merged**